src/components/ProductTable/MobileFilters.tsx

```diff
@@ -69,7 +69,7 @@ const MobileFilters = ({
             <p>{t("table-filters")}</p>
             <p className="text-body-medium">{` ${activeFiltersCount} ${t("table-active")}`}</p>
           </div>
-          <FilterBurgerIcon className="h-8 w-8 stroke-primary" />
+          <FilterBurgerIcon className="broke h-8 w-8 stroke-primary" />
```

Comment:
what is `broke` for?

Reply:
That must've been a wacky typo/mis-key combo! 😆 I'll remove it.